dask/dataframe/io/parquet/utils.py

```diff
@@ -611,6 +611,12 @@ def _aggregate_stats(
         return {}
     elif len(file_row_group_column_stats) == 0:
         assert len(file_row_group_stats) == 1
+        for j, stats in enumerate(file_row_group_stats[0]["columns"]):
+            if stats.get("max") == stats.get("min") and stats.get("null_count"):
+                # Remove "dangerous" stats (min == max, but null values exist)
+                file_row_group_stats[0]["columns"][j] = {
+                    "null_count": stats.get("null_count")
+                }
```

Comment:
This seems to be a case we're taking special care to account for. Is there a test that cover this?

Reply:
Yes, both new tests fail without this new logic.  However, it *may* make sense to remove this and simply check the null count when min=max in `apply_filters` (I'll check if that is cleaner).

Reply:
I ended up moving this logic to apply_filters